PublicMapping/districtbuilder
# Consistent button group styling

## Overview

Adds consistent button group styling in the map header.

### Checklist


### Demo

![Screen Shot 2020-09-22 at 5 16 34 PM](https://user-images.githubusercontent.com/1809908/93938668-94b17180-fcf7-11ea-8450-d5cf77c0e9d7.png)

### Notes

A reason these button styles get complicated is that for the geounit buttons, we need to wrap those in `span` elements to allow us to add a tooltip on a disabled button. Without the `span`, the tooltip would not fire.

## Testing Instructions

- Open a project
- Confirm the two button groups have similar styling applied

Closes #438
